sandworm/lib/web.ml

```diff
@@ -1,63 +1,68 @@
-open Tyxml.Html
+module T = Tyxml.Html
 
-let bucket_url path =
-  Format.sprintf "https://dune-binary-distribution.s3.fr-par.scw.cloud/%s" path
-;;
-
-let title = title (txt "Dune Dev Preview")
-let css = link ~rel:[ `Stylesheet ] ~href:"main.css" ()
+let bucket_url ~url path = Format.sprintf "%s/%s" url path
+let title = T.title (T.txt "Dune Dev Preview")
+let css = T.link ~rel:[ `Stylesheet ] ~href:"main.css" ()
 
 let plausible_js =
-  script
+  T.script
     ~a:
-      [ Tyxml.Html.Unsafe.string_attrib "data-domain" "dune.ci.dev"
-      ; a_defer ()
-      ; a_src (Xml.uri_of_string "https://plausible.ci.dev/js/script.file-downloads.js")
+      [ T.Unsafe.string_attrib "data-domain" "dune.ci.dev"
+      ; T.a_defer ()
+      ; T.a_src
+          (T.Xml.uri_of_string "https://plausible.ci.dev/js/script.file-downloads.js")
       ]
-    (txt "")
+    (T.txt "")
 ;;
 
 let headers = [ css ]
-let main_title = h2 [ txt "Dune distribution binary" ]
+let main_title = T.h2 [ T.txt "Dune binary distribution" ]
 
 let install =
-  [ p
-      [ txt
-          "After downloaded and extracted, you could use the command below to install it."
+  [ T.p
+      [ T.txt "First, download the dune binary associated with your system requirement." ]
+  ; T.p
+      [ T.txt
+          "Then, you can install Dune by running the following command from the location \
+           where you downloaded the executable file:"
```

Comment:
Are we not compressing the binary at all and just distributing the binary file directly?

Reply:
This is what we are currently doing, but we can move to a compressed version if we want to. What do you think?

Reply:
Compressing adds additional steps like e.g. needing a gzip, bzip2, xz or pkzip implementation. Unless traffic turns out to be an issue or users complain about the size I wouldn't bother.

Also makes it easier to know what the checksum is on: the binary not the compressed file.